model/project.go

```diff
@@ -261,23 +262,88 @@ type PluginCommandConf struct {
 	// TimeoutSecs indicates the maximum duration the command is allowed to run for.
 	TimeoutSecs int `yaml:"timeout_secs,omitempty" bson:"timeout_secs"`
 
-	// Params are used to supply configuratiion specific information.
-	Params map[string]interface{} `yaml:"params,omitempty" bson:"params"`
+	// Params are used to supply configuration specific information.
+	// map[string]interface{} is stored as a JSON string.
+	Params map[string]interface{} `yaml:"-" bson:"-"`
```

Comment:
Future investigation, I think we should not remove this, because there are a few places that we assign new values to Params (not in tests), and it seems wrong to marshal it into a YAML string at that point.